templates/typescript/app/views/barAggregated.ts

```diff
@@ -26,4 +26,5 @@ export const BarAggregatedMV = new MaterializedView<BarAggregated>({
   FROM ${barTable}
   GROUP BY toDayOfMonth(utcTimestamp)
   `,
+  selectTables: [barTable],
```

Comment:
we're already referencing the columns from `barTable` in the `sql`, having to specify this again feels a bit repetitive.

but then this is explicit and matches the python api

Reply:
Yeah - I am thinking we do it explicit for now and we improve the api after.
the select can also be a string that would be to be parsed and we would need to extract the table.
I would rather do that in a follow up improvement to the feature.